### PR TITLE
WIP Sandra/helppage

### PR DESCRIPTION
**Pull request by Sjones352**
Connect #1430

**Screenshot**
screen shot 2017-04-18 at 10 20 21 am

### CODE REVIEW

File: app/views/help_dispatch/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,135 @@
+
+<div class="cf-app-segment cf-app-segment--alt cf-help-content">
+  <h1 id="#top">Welcome to the Dispatch Help Page!</h1>
+  
+  <p>Here you will find <font color="#0872b9"><a href="#training-videos">Training Videos </font></a>and<font color="#0872b9"><a href="#faq"> FAQs </font></a>to Dipatch. If you still need help please contact the Caseflow Product Support Team by calling 1-844-876-5548. We will assist you as quickly as possible. </p>
```
Comment (mmlumba) [inline app/views/help_dispatch/show.html.erb]: You should create a CSS class for these types of links and apply the class instead. Add the classes to `help_dispatch.scss`

File: app/views/help_dispatch/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  <div class="cf-help-divider"></div>
+
+     
+  <ul id="toc" class="usa-unstyled-list">
+    <li><a href="#what-is-caseflow-dispatch">1. What is Caseflow Dispatch (or Dispatch)?</a></li>
+    <li><a href="#what-does-caseflow-dispatch-do">2. How was Dispatch developed? Who was involved?</a></li>
+    <li><a href="#how-to-access">3. How do I access Dispatch?</a></li>
+    <li><a href="#claims-assigned">4. How are claims assigned to me?</a></li>
+    <li><a href="#how-many-claims">5. How do I know how many claims I have completed?</a></li>
+    <li><a images/cancel-process">12. How can I cancel the process?</a></li>
```
Comment (mmlumba) [inline app/views/help_dispatch/show.html.erb]: possibly missing a `"` here

**Comment by mmlumba** [comment]
Hey Sandra, please reference the issue # in your ticket like so:
"connects #<issue number>"

File: app/views/help_dispatch/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  <p>
+    Dispatch automatically assigns you a pre-set number of claims for the day and routes the next claim in your station's work queue to you as you complete claims. Click "establish claim" to start the next claim in the queue. 
+
+  <h2 id="how-many-claims">5. How do I know how many claims I have completed?</h2>
+
+  <p>
+     The work history section provides a list of the claims that you have processed that day. 
+  </p>
+
+  <h2 id="veteran-information">6. Where does the Veteran information in the End Product (EP) form come from?</h2>
```
Comment (mmlumba) [inline app/views/help_dispatch/show.html.erb]: Questions 6-11 don't appear in the bulleted list above. Is this intentional?

File: spec/controllers/help_dispatch_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
```
Comment (mmlumba) [inline spec/controllers/help_dispatch_controller_spec.rb]: You don't really need a controller spec for this. You can create a feature test that shows that the user is on the help page in `spec/feature/`. Take a look at `spec/feature/help_spec.rb` for an example.

File: app/helpers/help_dispatch_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module HelpDispatchHelper
```
Comment (mmlumba) [inline app/helpers/help_dispatch_helper.rb]: Why are you adding a `Helper` module here? If there is some code that is shared between your controller and your view you can add the `:helper_method` symbol for the methods you want to be used in the view.

File: spec/helpers/help_dispatch_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
```
Comment (mmlumba) [inline spec/helpers/help_dispatch_helper_spec.rb]: Again....not exactly sure what you need the helper for or why you're testing it.

Comment (Sjones352) [inline spec/helpers/help_dispatch_helper_spec.rb]: Took the file out

File: app/helpers/help_dispatch_helper.rb (hunk 5 above)
Comment (Sjones352) [inline app/helpers/help_dispatch_helper.rb]: Removed this file

File: spec/controllers/help_dispatch_controller_spec.rb (hunk 4 above)
Comment (Sjones352) [inline spec/controllers/help_dispatch_controller_spec.rb]: Removed this file

File: app/views/help_dispatch/show.html.erb (hunk 3 above)
Comment (Sjones352) [inline app/views/help_dispatch/show.html.erb]: Added more FAQs

File: app/views/help_dispatch/show.html.erb (hunk 2 above)
Comment (Sjones352) [inline app/views/help_dispatch/show.html.erb]: Corrected this

File: app/views/help_dispatch/show.html.erb (hunk 1 above)
Comment (Sjones352) [inline app/views/help_dispatch/show.html.erb]: Actually removed the fonts because these are links

File: app/assets/javascripts/help_dispatch.js [hunk 7]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (Sjones352) [inline app/assets/javascripts/help_dispatch.js]: Removed this

File: app/assets/stylesheets/help_dispatch.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the help_dispatch controller here.
```
Comment (Sjones352) [inline app/assets/stylesheets/help_dispatch.scss]: These are actually links so removed these

File: config/routes.rb [hunk 9]
```diff
@@ -81,10 +81,12 @@
   get "styleguide", to: "styleguide#show"
 
   get 'help' => 'help#show'
+  get 'help/dispatch' => 'help#dispatch'
```
Comment (mmlumba) [inline config/routes.rb]: Nice routing!

**Pull request merged**